components/brave_new_tab_ui/components/default/notification/index.ts

```diff
@@ -9,13 +9,12 @@ export const SiteRemovalNotification = styled<{}, 'header'>('header')`
   border-radius: 8px;
   box-shadow: 2px 2px 6px rgba(0,0,0,0.3);
   background-color: #fff;
-  width: 100%;
-  max-width: 500px;
   height: 100px;
   align-items: center;
   padding: 30px 60px;
-  display: flex;
-  justify-content: space-between;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
```

Comment:
```suggestion
 grid-auto-flow: column;
 grid-auto-columns: auto;
```

nit: if you make this change then you won't need to explicitly specify the exact amount of columns, and have this break if more children are added.

Reply:
@cezaraugusto I accepted this proposed change and merged to get a Nightly build started with the fix, as this is a bug in release channel

If you disagree with the suggested change or needed to make any other tweaks, let's do a follow up 😄